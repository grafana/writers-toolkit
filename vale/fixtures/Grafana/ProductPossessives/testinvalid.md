Don't use:

- Grafana's
- Grafanaâ€™s
