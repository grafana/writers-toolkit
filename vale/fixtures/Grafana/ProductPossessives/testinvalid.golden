testinvalid.md:3:3:Grafana.ProductPossessives:Don't form a possessive from a feature name, product name, or trademark, regardless of who owns it. Instead, use the name as a modifier or rewrite to use a word like of to indicate the relationship. 
testinvalid.md:4:3:Grafana.ProductPossessives:Don't form a possessive from a feature name, product name, or trademark, regardless of who owns it. Instead, use the name as a modifier or rewrite to use a word like of to indicate the relationship. 
